src/test/regress/expected/columnar_paths.out

```diff
@@ -4,6 +4,10 @@ CREATE TABLE full_correlated (a int, b text, c int, d int) USING columnar;
 INSERT INTO full_correlated SELECT i, i::text FROM generate_series(1, 1000000) i;
 CREATE INDEX full_correlated_btree ON full_correlated (a);
 ANALYZE full_correlated;
+-- Prevent qual pushdown from competing with index scans. This test
+-- file tests paths other than qual pushdown; those are tested in
+-- columnar_chunk_filtering.sql.
+SET columnar.enable_qual_pushdown = false;
```

Comment:
since now this file has some tests where `columnar.enable_qual_pushdown` is set to `DEFAULT`:

```suggestion
-- Prevent qual pushdown from competing with index scans.
SET columnar.enable_qual_pushdown = false;
```